test/support/test_complex.h

```diff
@@ -86,14 +86,27 @@ run_test()
 //         // ...
 //     }
 #define IF_DOUBLE_SUPPORT(x)                                                                          \
-    if constexpr (HasDoubleSupportInRuntime::value) { x; }
+    TestUtils::invoke_test_if<>(HasDoubleSupportInRuntime(), [](){ x; });
 
 // We should use this macros to avoid compile-time error in code with long double type in Kernel.
 #define IF_LONG_DOUBLE_SUPPORT(x)                                                                     \
-    if constexpr (HasLongDoubleSupportInCompiletime::value) { x; }
+    TestUtils::invoke_test_if<>(HasLongDoubleSupportInCompiletime(), []() { x; });
```

Comment:
We have the same helpers in OneDPL "utils.h" - __invoke_if, __invoke_if_else, __invoke_if_not

Reply:
I think that I haven't ablility to use these helper classes from include\oneapi\dpl\pstl\utils.h in tests.

Reply:
Correct, using internal implementation details of the library is generally not appropriate for tests.